soc/arm/gigadevice/gd32e10x/Kconfig.defconfig.series

```diff
@@ -0,0 +1,11 @@
+# Copyright (c) 2021 YuLong Yao <feilongphone@gmail.com>
+# SPDX-License-Identifier: Apache-2.0
+
+if SOC_SERIES_GD32E103
+
+source "soc/arm/gigadevice/gd32e10x/Kconfig.defconfig.gd32*"
+
+config SOC_SERIES
+	default "gd32e10x"
+
```

Comment:
this is inconsistent with `SOC_SERIES_GD32E103`, you likely need `SOC_SERIES_GD32E10X`

Reply:
Change all `SOC_SERIES_GD32E103` to `SOC_SERIES_GD32E10X`.